src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
+
 export interface Figure {
+  color: Color,
+  shape: Shape,
 
+  getArea(): number,
 }
 
-export class Triangle {
-
+function isCompare(a: number, b: number, c: number): boolean {
+  return a + b <= c || b + c <= a || c + a <= b;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = 'triangle' as Shape;
+
+  color: Color;
+
+  constructor(
+    color: Color,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    this.color = color;
```

Comment:
You can declare color with 'private' or 'public' keywords, so you dont need to declare it in constructor body.

Reply:
Good point, fix also in other places